cmd/allocator/metrics.go

```diff
@@ -65,6 +71,8 @@ func parseEnvFlags() config {
 	viper.SetDefault(remoteAllocationTimeoutFlag, 10*time.Second)
 	viper.SetDefault(totalRemoteAllocationTimeoutFlag, 30*time.Second)
 
+	pflag.Int32(apiServerSustainedQPSFlag, 100, "Maximum sustained queries per second to send to the API server")
+	pflag.Int32(apiServerBurstQPSFlag, 200, "Maximum burst queries per second to send to the API server")
```

Comment:
viper.GetInt(apiServerBurstQPSFlag) to follow the pattern?
Is there any case that the defaults are used? Why not set it to the helm default?

Reply:
No reason - I literally just copied exactly what was in the controller/main.go - figured might as well keep it all consistent.
https://github.com/googleforgames/agones/blob/master/cmd/controller/main.go#L258

Reply:
I think we should use the same default across stack, if the default values are not meant to be different, regardless of the controller. 

If we set the default to 100 here and 400 as the environment variable, I can imagine looking at the code, one may assume the default is 100 as they have not set that on the environment variable.

Reply:
I also thought this was confusing, but it's consistent with the agones controller. Maybe a cleanup there would be better than propagating confusing dueling default values?

Reply:
Do we feel we should cleanup here, or do it in a subsequent PR? I don't mind either way tbh.

Reply:
For allocator, because we are making the change in this PR, we should fix it in this PR. For controller, we can fix it in a later PR.

Reply:
Thought I hit this, and apparently I missed it. Fix incoming!